src/Common/src/Interop/Windows/mincore/Interop.QuerySystemInformation.cs

```diff
@@ -0,0 +1,54 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Text;
+
+internal partial class Interop
+{
+    internal partial class mincore
+    {
+        internal enum SYSTEM_INFORMATION_CLASS
+        {
+            SYSTEM_PROCESSOR_PERFORMANCE_INFORMATION = 8
+        }
+
+        internal struct SYSTEM_PROCESSOR_PERFORMANCE_INFORMATION
+        {
+            // These fields are all assigned by the native call.
+#pragma warning disable CS0649
```

Comment:
Mark the struct as `[StructLayout(LayoutKind.Sequential)]`. It will disable the warning as well, but it looks better.